dotnet/Razorvine.Pyrolite/Pyrolite/Pickle/PickleUtils.cs

```diff
@@ -288,7 +284,33 @@ public static string rawStringFromBytes(byte[] data)
         }
         return result;
 	}
-	
+
+    internal static int CheckedCast(long length)
```

Comment:
I'm surprised this got separated out into its own method, given your comments about wanting to eek out every last bit of perf.  This will not be inlined because of the exception handling.

Reply:
good point, I forgot about it.